### ISSUE TITLE
Reverse Proxy

### ISSUE THREAD

**Issue by borag** [CrowdHailer/raxx_kit#101]
Is there a single-port reverse proxy sample like master _proxy (https://github.com/wojtekmach/acme_bank/tree/master/apps/master_proxy) ?

Trying to find raxx_cowboy to adapt but failing to find a repo named like that.

**Comment by CrowdHailer** [comment]
I don't know of one

**Comment by borag** [comment]
Don't want to use Traefik like non-erlang solutions.
Digging in :)

**Comment by borag** [comment]
Where is raxx_cowboy repo ? Deleted ?

**Comment by CrowdHailer** [comment]
Yes, raxx_cowboy is no longer supported by me. Other people where looking into it. You might have some luck asking in the Raxx channel on the Elixir slack
